paymentsheet/src/main/java/com/stripe/android/paymentsheet/addresselement/AddressLauncher.kt

```diff
@@ -200,20 +200,18 @@ class AddressLauncher internal constructor(
             REQUIRED
         }
     }
+}
 
-    companion object {
-        @Composable
-        fun rememberLauncher(
-            callback: AddressLauncherResultCallback
-        ): AddressLauncher {
-            val activityResultLauncher = rememberLauncherForActivityResult(
-                contract = AddressElementActivityContract(),
-                onResult = callback::onAddressLauncherResult
-            )
+@Composable
+fun rememberAddressLauncher(
```

Comment:
Can we add documentation here? (second line due to usage of `rememberLauncherForActivityResult`)

```
Creates an [AddressLauncher] that is remembered across compositions.

This *must* be called unconditionally, as part of the initialization path.
```